modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/classloader/ArtifactClassLoaderResolverConstants.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.activation.internal.classloader;
+
+import static org.mule.runtime.container.api.ContainerClassLoaderProvider.createContainerClassLoader;
+
+import org.mule.runtime.container.api.ModuleRepository;
+import org.mule.runtime.container.internal.ContainerModuleDiscoverer;
+import org.mule.runtime.container.internal.DefaultModuleRepository;
+import org.mule.runtime.module.artifact.activation.api.classloader.ArtifactClassLoaderResolver;
+import org.mule.runtime.module.artifact.api.classloader.ArtifactClassLoader;
+
+/**
+ * Provides constants to be used internally for the generation of classloaders
+ *
+ * @since 4.5
+ */
+public class ArtifactClassLoaderResolverConstants {
+
+  /**
+   * A {@link ModuleRepository} for the modules available in the same classpath as this class.
```

Comment:
Shouldn't the class loader of `ArtifactClassLoaderResolverConstants` be used instead of `ArtifactClassLoaderResolver` for this to be strictly true?